Microsoft.Toolkit.Uwp/Helpers/HttpHelper/HttpHelper.cs

```diff
@@ -104,6 +109,45 @@ public HttpHelper(int poolSize, IHttpFilter httpFilter)
             }
         }
 
+        public async Task<HttpHelperResponse> GetRequestAsync(HttpHelperRequest request, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
SendRequestAsync will not work. I think Twitter service should instantiate its own instance of HttpHelper rather than using HttpHelper.Instance.

Each request should clear the Default Headers previously set just in case.